projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -48,8 +48,9 @@ public function init() {
 			return;
 		}
 
+		$in_jetpack = ( class_exists( 'Jetpack' ) && method_exists( 'Jetpack', 'enable_module_configurable' ) ) ? true : false;
 		// Assets (css, js).
-		if ( $this->in_jetpack ) {
```

Comment:
I can see this gets set in the constructor of `Jetpack_Publicize`. What about this change means we need to recalculate it here?

Reply:
The original reason why I had to recalculate was that `$in_jetpack` is not available as a part of the Publicize_UI object. That being said, I think there will be a scenario in the future where this won't work within the stand-alone plugin. 

We are calling the `load_assets` right after this line, which anyway checks if the `Jetpack_Admin_Page` class exists before proceeding further. So, I think should be safe to remove this  `in_jetpack` check altogether,  let me know if you agree with this @pablinos.